### PR TITLE
RATEST-282: Modify user account test to generate different username on each run

### PR DESCRIPTION
**Pull request by kdaud**
Ticket [ID](https://issues.openmrs.org/browse/RATEST-282)

Description -> Modify user account test to generate different username on n each run

### CODE REVIEW

File: qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageUserAccountPage.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
 	private static final By GIVEN_NAME = By.cssSelector("#adminui-givenName-field");
 	private static final By GENDER_FEMALE = By.cssSelector("#adminui-gender-1-field");
 	private static final By ADD_USER_ACCOUNT = By.id("adminui-addUserAccount");
-	private static final By USER_NAME = By.cssSelector("#adminui-username-field");
+	private static final By USERNAME_FIELD = By.cssSelector("#adminui-username-field");
```
Comment (sherrif10) [inline qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageUserAccountPage.java:26]: How do you generate different username here.

File: qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java [hunk 2]
```diff
@@ -28,6 +29,9 @@
 public class UserAccountSteps extends Steps {
 
 	private static final String SYSTEM_ALERT = "Atleast 8 character(s) are required";
+	private static final String CLERK = RandomStringUtils.randomAlphabetic(8);
```
Comment (sherrif10) [inline qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java:32]: How does this attribute work in specific terms. how does it generate different username

File: qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java [hunk 3]
```diff
@@ -28,6 +29,9 @@
...
+	private static final String NURSE = RandomStringUtils.randomAlphabetic(8);
+	private static final String DOCTOR = RandomStringUtils.randomAlphabetic(8);
```
Comment (sherrif10) [inline qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java:34]: Ohh cool i can see its being used [here](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/master/ui-tests/src/test/java/org/openmrs/reference/AddVisitTypeTest.java#L31). perhaps it could be a great refactor.

File: qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageUserAccountPage.java (hunk 1 above)
Comment (kdaud) [reply qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageUserAccountPage.java:26]: The param passed in [setUsername()](https://github.com/openmrs/openmrs-contrib-qaframework/blob/5bb1758a8737c188e79e14f2250fe554b042026d/qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageUserAccountPage.java#L77) during its call is a randomly generated string.

File: qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java (hunk 2 above)
Comment (kdaud) [reply qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java:32]: For each run RandomStringUtils.randomAlphabetic() generates a unique String that is passed when setting the [username](https://github.com/openmrs/openmrs-contrib-qaframework/blob/5bb1758a8737c188e79e14f2250fe554b042026d/qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java#L83)

File: qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageUserAccountPage.java [hunk 4]
```diff
@@ -74,9 +74,8 @@ public void clickOnAddUserAccount() {
 		clickOn(ADD_USER_ACCOUNT);
 	}
 
-	public void enterUsername(String username) {
-		findElement(USER_NAME).clear();
-		findElement(USER_NAME).sendKeys(username);
```
Comment (dkayiwa) [inline qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageUserAccountPage.java:79]: How is this change related to what you are fixing?

Comment (kdaud) [reply qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageUserAccountPage.java:79]: The method enterUsename() whenever invoked has been using a fixed string so have modified it to accept a random string.

Comment (kdaud) [reply qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageUserAccountPage.java:79]: By the way retaining the method the way it was also does the work.
